web/secure_css_wrapper.php

```diff
@@ -29,6 +29,7 @@
 $content = file_get_contents($fileUri);
 
 // replace links to http://www.wikidot.com/... and to the current host.
+# XXX mix-up between http and https?
```

Comment:
No, this is checking whether the wiki is set to be secure-only and is rewriting a typical HTTP location to HTTPS on the fly.